### PR TITLE
chore: use newTokens for search [B1M2-5050]

### PR DESCRIPTION
**Pull request by alexisdevtailor**

### CODE REVIEW

**Review by dkireev — CHANGES_REQUESTED** [review]
some properties have numbers where should be pixels. Somewhere old tokens are still in use

**Review by dkireev — CHANGES_REQUESTED** [review]
settings applied without rem function

File: packages/core/src/search/private/components/Input.ts [hunk 1]
```diff
@@ -83,69 +83,55 @@ export const SearchInput = styled.input<
...
+  textOverflow: 'ellipsis',
+  overflow: 'hidden',
+  backgroundColor: colorNew.goku,
+  color: colorNew.trunks,
+  borderRadius: newTokens.borderRadius.medium,
+  border: hasBorder ? `${rem(borderWidth)} solid transparent` : 'none',
+  outline: 0,
+  zIndex: zIndex.searchInput,
+  transition: `${inputConfig.transition}`,
+  transitionDuration: newTokens.transitionDuration,
```
Comment (dkireev) [inline packages/core/src/search/private/components/Input.ts:105]: There is no transitionDuration token anymore

**Review by dkireev — CHANGES_REQUESTED** [review]

File: packages/core/src/search/private/components/Input.ts [hunk 2]
```diff
@@ -83,48 +83,56 @@ export const SearchInput = styled.input<
...
+    textOverflow: 'ellipsis',
+    overflow: 'hidden',
+    backgroundColor: colorNew.goku,
+    color: colorNew.trunks,
+    borderRadius: newTokens.borderRadius.medium,
+    border: hasBorder ? `${rem(borderWidth)} solid transparent` : 'none',
+    outline: 0,
+    zIndex: zIndex.searchInput,
+    transition: `${inputConfig.transition}`,
+    transitionDuration,
```
Comment (dkireev) [inline packages/core/src/search/private/components/Input.ts:110]: No, it's an old token. Either replace it with a hardcode, or re-think transition and transitionDuration

**Review by dkireev — CHANGES_REQUESTED** [review]

**Comment by karl-kallavus** [comment]
@alexisdevtailor please update tests

**Review by karl-kallavus — CHANGES_REQUESTED** [review]
Please update tests

**Review by dkireev — APPROVED** [review]

**Pull request merged**